### ISSUE TITLE
[BUG] QuickCreate Save doesn't work after navigating from another quick create

### ISSUE THREAD

**Issue by devonuto** [Microsoft/EasyRepro#891]
## Bug Report

**EasyRepro Version**
Microsoft Dynamics 365 Online Version 9.1 (9.1.x) (DB 9.1.x) online
  
**UCI or Classic Web**
UCI (XrmApp)

**Online or On Premise**
Online

**Browser**
chrome=83.0.4103.97

**Describe the bug**  
When dealing with a quick create form which was created from another Quick Create form, executing XrmApp.QuickCreate.Save() results in this error:

OpenQA.Selenium.ElementClickInterceptedException: 'element click intercepted: Element <button type="button" id="quickCreateSaveAndCloseBtn" data-id="quickCreateSaveAndCloseBtn" accesskey="S" aria-label="Save and Close" tabindex="0" class="ms-Button ms-Button--primary root-133" data-is-focusable="true" style="border-color: transparent; background-color: rgb(59, 121, 183); color: rgb(255, 255, 255); height: 2.5rem; min-width: 6rem; cursor: pointer; justify-content: center; align-items: center; padding-left: 1rem; padding-right: 1rem; margin-top: 0px; margin-bottom: 0px; margin-left: 0px; font-size: 14px; font-style: normal; font-family: SegoeUI, &quot;Segoe UI&quot;; width: auto;">...</button> is not clickable at point (1710, 886). Other element would receive the click: <div class="ms-Button-label label-137" id="id__10">...</div>

xrmApp.Navigation.QuickCreate("contact") also errors from the Case quick create form:
OpenQA.Selenium.ElementClickInterceptedException: 'element click intercepted: Element <button aria-hidden="true" tabindex="-1" data-id="quickCreateLauncher" data-lp-id="global-quickCreateLauncher" type="button" class="t u dz v ea j r s ad ae af flexbox">...</button> is not clickable at point (1703, 24). Other element would receive the click: <div role="presentation" class="a qi bb i xw flexbox">...</div>
  (Session info: chrome=83.0.4103.97)'

**Code to reproduce**  
From Email entity:

xrmApp.CommandBar.ClickCommand("Create Case");
xrmApp.QuickCreate.SetValue("new_entryid", 12345);
// can't use xrmApp.Navigation.QuickCreate("contact"); so I do this:
xrmPage.Browser.Driver.WaitUntilAvailable(By.XPath("//button[contains(@data-id,'new_contact')]"));
xrmPage.Browser.Driver.FindElement(By.XPath("//button[contains(@data-id,'new_contact')]")).Click(true);
xrmPage.Browser.Driver.WaitUntilAvailable(By.XPath("//button[contains(@aria-label,'New Contact')]"));
xrmPage.Browser.Driver.FindElement(By.XPath("//button[contains(@aria-label,'New Contact')]")).Click(true);

xrmApp.QuickCreate.SetValue("firstname", DataConstants.contactFirstName);
xrmApp.QuickCreate.SetValue("lastname", DataConstants.contactLastName);
xrmApp.QuickCreate.SetValue("emailaddress1", DataConstants.contactEmail);
xrmApp.QuickCreate.Save(); -- errors here
 
**Expected behavior**  
I expect it to save

**Additional context**  
The error suggests the click is blocked by the ms-Button-label.  So I tried this to see if I can click on the label instead, and found there are two identical labels rendered on the page:

var quickCreate = xrmPage.Browser.Driver.FindElement(By.XPath("//section[@data-id=\"quickCreateRoot\" and contains(@data-lp-id, \"contact\")]"));
var button = quickCreate.FindElement(By.XPath("//button[@id=\"quickCreateSaveAndCloseBtn\"]"));
button.FindElements(By.XPath("//div[contains(@class, \"ms-Button-label\") and text()=\"Save and Close\"]"))
Count = 2
    [0]: {Element (id = 772d6274-9c6d-4d76-a196-c3b1aebd98e4)}
    [1]: {Element (id = cecac6bb-74f1-4f7f-9c4e-ce5ba5d444a3)}
button.FindElements(By.XPath("//div[contains(@class, \"ms-Button-label\") and text()=\"Save and Close\"]"))[0]
{Element (id = 772d6274-9c6d-4d76-a196-c3b1aebd98e4)}
    Coordinates: {OpenQA.Selenium.Remote.RemoteCoordinates}
    Displayed: true
    Enabled: true
    Id: "772d6274-9c6d-4d76-a196-c3b1aebd98e4"
    Location: {X = 1665 Y = 880}
    LocationOnScreenOnceScrolledIntoView: {X = 1665 Y = 880}
    Selected: false
    Size: {Width = 93 Height = 14}
    TagName: "div"
    Text: "Save and Close"
    WrappedDriver: {OpenQA.Selenium.Chrome.ChromeDriver}
button.FindElements(By.XPath("//div[contains(@class, \"ms-Button-label\") and text()=\"Save and Close\"]"))[1]
{Element (id = cecac6bb-74f1-4f7f-9c4e-ce5ba5d444a3)}
    Coordinates: {OpenQA.Selenium.Remote.RemoteCoordinates}
    Displayed: true
    Enabled: true
    Id: "cecac6bb-74f1-4f7f-9c4e-ce5ba5d444a3"
    Location: {X = 1665 Y = 880}
    LocationOnScreenOnceScrolledIntoView: {X = 1665 Y = 880}
    Selected: false
    Size: {Width = 93 Height = 14}
    TagName: "div"
    Text: "Save and Close"
    WrappedDriver: {OpenQA.Selenium.Chrome.ChromeDriver}

Clicking the second element will get the quick create to save.  The first one will not.

**Comment by shariquekhan1997** [comment]
You have to use SwitchTo() method to switch from one form to another then you can click on the buttons on the form you switched to. The issue is which is causing you is because the target window is the previous form and you are clicking on another form.
@devonuto

**Comment by devonuto** [comment]
> You have to use SwitchTo() method to switch from one form to another then you can click on the buttons on the form you switched to. The issue is which is causing you is because the target window is the previous form and you are clicking on another form.
> @devonuto

Given SwitchTo is a Selenium WebDriver function, it's not exactly a EasyRepro fix.

I was able to code around it using Selenium, but it still remains a bug with EasyRepro.

**Comment by shariquekhan1997** [comment]
> > You have to use SwitchTo() method to switch from one form to another then you can click on the buttons on the form you switched to. The issue is which is causing you is because the target window is the previous form and you are clicking on another form.
> > @devonuto
> 
> Given SwitchTo is a Selenium WebDriver function, it's not exactly a EasyRepro fix.
> 
> I was able to code around it using Selenium, but it still remains a bug with EasyRepro.

Yes I provided you the workaround that I also used. I am also an end-user like you. 😊
